### PR TITLE
Payment Request throws if PMIs is invalid

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None



View more information about this build on:

- [WPT Results Dashboard](http://pulls.web-platform-tests.org/build/17250)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/263347650)

**Comment by marcoscaceres** [comment]
Rebased

**Comment by sideshowbarker** [comment]
Chrome build is hitting the “No output has been received in the last 10m0s” Travis timeout

https://travis-ci.org/w3c/web-platform-tests/jobs/264234850

**Comment by marcoscaceres** [comment]
@sideshowbarker thanks, restarted that job again... let's see what happens.

**Review by sideshowbarker — APPROVED** [review]
Reviewed and LGTM but feel free if you want to wait to merge until you also get implementor review

**Comment by marcoscaceres** [comment]
Thanks @sideshowbarker - bounced to @edenchuang for review.

File: payment-request/payment-request-ctor-pmi-handling.https.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      supportedMethods: "http://username:password@example.com/pay",
+    },
+    {
+      supportedMethods: "http://foo.com:100000000/pay",
+    },
+    {
+      supportedMethods: "basic-💳",
+    },
+    {
+      supportedMethods: "https://wpt.fyi/payment-request",
```
Comment (edenchuang) [inline payment-request/payment-request-ctor-pmi-handling.https.html]: This seems to be a valid supportedMethods.

**Review by edenchuang — COMMENTED** [review]
Need an invalid supportedMethods for "Basic-Card" case.

**Review by edenchuang — APPROVED** [review]
LGTM.

**Pull request merged**